bin/install.sh

```diff
@@ -96,24 +96,23 @@ install_foss_server() {
     echo -n "$(install_depot_server force)"/foss
 }
 
-install_pip_install() {
-    # --no-color does not work always
-    # --progress-bar=off seems to work
-    # but this seems to work always
-    pip install "$@" | cat
-}
-
-install_pip_uninstall() {
-    # we don't care if uninstall work
-    pip uninstall -y "$@" >& /dev/null || true
-}
-
-install_proprietary_server() {
-    # proprietary is best served from the $install_server, which
-    # will be the local server (dev). Having a copy of the code
-    # locally in dev is better than sharing the proprietary
-    # key in dev.
-    echo -n "$(install_depot_server)/$install_proprietary_key"
+install_git_clone() {
+    # repo of the form repo, org/repo or https://git-server/org/repo{.git,}
+    # You can set RADIA_RUN_GIT_CLONE_BRANCH_<REPO> to clone a specific branch.
+    # RADIA_RUN_*  carries inside containers and ssh (see install_vars_export).
+    declare repo=$1
+    declare b=${repo##*/}
+    if [[ $b == $repo ]]; then
+        repo=radiasoft/$repo
+    fi
+    if [[ ! $repo =~ ^https? ]]; then
+        repo=https://github.com
```

Comment:
should this be `repo=https://github.com/$repo`?

Reply:
@robnagler ping

Reply:
https://github.com/radiasoft/download/pull/678